cli/pcluster/dcv/connect.py

```diff
@@ -38,7 +38,7 @@ def _check_command_output(cmd):
     return sub.check_output(cmd, shell=True, universal_newlines=True, stderr=sub.STDOUT).strip()
 
 
-def dcv_connect(args):
+def dcv_connect(args, extra_args):
```

Comment:
We have to remember in future to change this, because must be possible to pass the extra_args only to the "dcv connect" subcommand not the others subcommands.